src/Layout/Presentation/SingleDistrict/SingleDistrict.tsx

```diff
@@ -116,7 +116,7 @@ export class SingleDistrict extends React.Component<SingleDistrictProps, {}> {
                         {
                             Header: <span className="is-pulled-right wrap" >Oppslutning %</span>,
                             id: "%",
-                            accessor: (d: PartyResult) => d.percentVotes,
+                            accessor: "percentVotes",
```

Comment:
This change is wrong unless percentVote already has had numberFormat applied to it. I would prefer numberFormat being applied in the view and not in the data.

Reply:
Number format is being applied in the line below:
```javascript
Cell: (row) => {
     return numberFormat(row.value, decimals);
},
```

Reply:
Ah, I should start going through things commit by commit, perhaps.